conf.py

```diff
@@ -63,9 +64,56 @@
     '.. |compass| replace:: MongoDB Compass',
     '.. |realm| replace:: realm',
     '.. |realms| replace:: realms',
+    '.. |realm-cli| replace:: ``realm-cli``',
     '.. _compass: https://docs.mongodb.com/compass'
 ])
 
+source_constants = {
+    'realm': 'realm',
+    'realms': 'realms',
+    'backend': 'MongoDB Realm',
+    'client-database': 'Realm Database',
+    'atlas': 'MongoDB Atlas',
+    'atlas-short': 'Atlas',
+    'backend': 'MongoDB Realm',
+    'backend-short': 'Realm',
+    'service': 'MongoDB Realm',
+    'service-short': 'Realm',
+    'realm': 'realm',
+    'sync': 'Realm Sync',
+    'sync-short': 'Sync',
+    'sync-protocol': 'Realm Sync Protocol',
+    'sync-protocol-short': 'Sync Protocol',
+    'instance': 'Realm app',
+    'instances': 'Realm apps',
+    'instance-short': 'app',
+    'instances-short': 'apps',
+    'client-database': 'Realm Database',
+    'schema': 'Realm Data Model',
+    'backend-schema': 'Realm Schema',
+    'frontend-schema': 'Realm Object Model',
+    'cli': '``realm-cli``',
+    'ui': 'Realm UI',
+    'function': 'Realm function',
+    'functions': 'Realm functions',
```

Comment:
Same for "T"riggers:
```suggestion
    'functions': 'Realm Functions',
```

Reply:
added!